src/test/java/mate/academy/dao/RoleDaoTest.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.dao.impl.RoleDaoImpl;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoTest extends AbstractTest {
+    private static final String ROLE = Role.RoleName.USER.name();
+    private RoleDao roleDao;
+
+    @BeforeEach
+    void setUp() {
+        roleDao = new RoleDaoImpl(getSessionFactory());
+    }
+
+    @Test
+    void save_Ok() {
```

Comment:
be careful with the rules of naming methods, each new word starts with a small letter, check it for all cases 
```suggestion
    void save_ok() {
```